src/components/navbar.js

```diff
@@ -0,0 +1,17 @@
+import { Link } from 'react-router-dom';
+import './navbar.css';
+
+const Navbar = () => (
+  <nav className="nav">
```

Comment:
- [ ] [optional] It would have been nice to increase the size of the navbar and its content, to make it even more visually appealing

Reply:
Okay, I will do that now